### PR TITLE
Support alarm tags

### PR DESCRIPTION
**Pull request by liqiangz**
Support alarm tags.

### CODE REVIEW

File: alarm.graphqls
Comment (wu-sheng) [inline alarm.graphqls:42]: If you are asking for this change, please submit another new issue, we need to notify UI side PMC members, and get their approval.

Comment (wu-sheng) [reply alarm.graphqls:42]: The old tag discussion is only related to notification, it makes sense to add this on about native/default UI. But still, the process is required.

Comment (liqiangz) [reply alarm.graphqls:42]: I will open a issue later.

Comment (liqiangz) [reply alarm.graphqls:42]: @wu-sheng  Related issue https://github.com/apache/skywalking/issues/6821

**Review by wu-sheng — APPROVED** [review]
Good to me, wait for confirmation of https://github.com/apache/skywalking/issues/6821

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**
